doc/guide/installation.rst

```diff
@@ -16,6 +17,11 @@ The easiest way to keep your suite updated is to use
     .. code-block:: js
 
         {
+            "require": {
+                "behat/behat": "~3.0",
+                "behat/mink-extension": "~2.0@dev",
```

Comment:
This isn't needed at all. `behat/mink-extension` is a dependency of this package.